disnake/user.py

```diff
@@ -308,6 +313,19 @@ def display_name(self) -> str:
         """
         return self.global_name or self.name
 
+    @property
+    def primary_guild(self) -> Optional[PrimaryGuild]:
```

Comment:
To make type checkers understand `Member.primary_guild` (which is equivalent to `Member._user.primary_guild` under the hood) , this still needs to be added here:
https://github.com/DisnakeDev/disnake/blob/263c02d50bc5e4ea9ab102095fc01d500233d7d2/disnake/member.py#L299-L300